pkgs/top-level/all-packages.nix

```diff
@@ -11943,10 +11943,9 @@ with pkgs;
 
   playbar2 = libsForQt5.callPackage ../applications/audio/playbar2 { };
 
-  playwright = with python3Packages; toPythonApplication playwright;
+  playwright = (callPackage ../development/web/playwright/driver.nix { }).playwright;
```

Comment:
the python package should not have squatted the top-level `playwright` attribute so I am onboard with the change but that needs mentioning in the release notes.
It's too late for breaking changes for 24.05 nixos/doc/manual/release-notes/rl-2405.section.md, and nixos/doc/manual/release-notes/rl-2411.section.md does not exist yet so looks like it will have to wait.

Reply:
Reverted the change. I think playwright-driver should be renamed into `playwright-core` and `playwright` should point at `@playwright/test`. Better to do in next PR.